uefi/src/table/system.rs

```diff
@@ -211,6 +211,23 @@ impl SystemTable<Boot> {
     /// abstractions provided by this crate, invoking this function will
     /// automatically disable them.
     ///
+    /// # Safety
```

Comment:
Maybe we should mention that all abstraction provided by uefi leveraging boot services, such as global allocators and loggers, will panic.

Otherwise, it might not be clear and people think they have to unregister those in some way.

Reply:
Updated the prior paragraph since it already discusses the helpers. I think only the allocator panics, the logger is just disabled so that calling it silently does nothing.